app/Entry.js

```diff
@@ -1,14 +1,21 @@
 /* eslint-disable react/display-name */
+import React, { useContext, useEffect } from 'react';
```

Comment:
these import changes were to just group third party dependencies, screens, and other imports a bit better. The only new import is `import ExposureNotificationContext from './ExposureNotificationContext`